homeassistant/components/miraie_ac/config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Config flow for MirAIe AC integration."""
+from __future__ import annotations
+
+import logging
+import re
+from typing import Any
+
+from py_miraie_ac import (
+    AuthException,
+    AuthType,
+    ConnectionException,
+    MirAIeAPI,
+    MobileNotRegisteredException,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONFIG_KEY_USER_ID, COUNTRY_CODE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONFIG_KEY_USER_ID): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect."""
+
+    login_id = data[CONFIG_KEY_USER_ID]
+    password = data[CONF_PASSWORD]
+
+    if not _validate_mobile_number(login_id):
+        raise ValidationError()
+
+    async with MirAIeAPI(
+        auth_type=AuthType.MOBILE, login_id=login_id, password=password
+    ) as api:
+        _LOGGER.debug("Initializing MirAIe API")
+        await api.initialize()
+        _LOGGER.debug("Connection successful!")
+
+        # Return info that you want to store in the config entry.
+        return {"title": f"MirAIe ({login_id})"}
```

Comment:
```suggestion
        return {"title": login_id}
```

No need to specify again the integrations name